public/js/cinode-recruitment-public.js

```diff
@@ -61,6 +64,10 @@ jQuery(document).ready(function ($) {
         } else {
           formData.append("companyAddressId", "");
         }
+        
+        if (availableFrom){
```

Comment:
always true, since it's a jquery selector I guess.

Reply:
By default, from backend receives 0, if user select date than is true.

Reply:
aye, so since it's declared like this; `const availableFrom = $("#availableFrom");` the if statement is not needed since it's always true, but whatever :)

Reply:
There I check if the field exist, and I append the value to post.

Reply:
What he is saying is, you probably intended to test for mateched elements, `if(availableFrom.length > 0){`.